.github/workflows/capsule-cypress.yml

```diff
@@ -129,7 +129,7 @@ jobs:
           CYPRESS_TRADING_TEST_VEGA_WALLET_PASSPHRASE: ${{ secrets.CYPRESS_TRADING_TEST_VEGA_WALLET_PASSPHRASE }}
           CYPRESS_SLACK_WEBHOOK: ${{ secrets.CYPRESS_SLACK_WEBHOOK }}
           CYPRESS_ETH_WALLET_MNEMONIC: ${{ secrets.CYPRESS_ETH_WALLET_MNEMONIC }}
-          CYPRESS_TEARDOWN_NETWORK_AFTER_FLOWS: false
+          CYPRESS_TEARDOWN_NETWORK_AFTER_FLOWS: true
```

Comment:
(blocking) leave it on 'false' please, we had issue with some PRs when wallet was being reset in token tests at the same time that Trading tests needed that wallet

Reply:
Sure, done